powertools-parameters/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>powertools-parent</artifactId>
+        <groupId>software.amazon.lambda</groupId>
+        <version>0.2.0-beta</version>
+    </parent>
+
+    <artifactId>powertools-parameters</artifactId>
+
+    <name>AWS Lambda Powertools Java library Parameters</name>
+
+    <description>
+        Set of utilities to retrieve parameters from Secrets Manager, SSM Parameter Store or DynamoDB
```

Comment:
✅

Reply:
should remove DynamoDB a it is not yet implemented...